.eslintignore

```diff
@@ -0,0 +1,42 @@
+build/
+node_modules/
+.github/
+.idea/
+api_mock/
+public/
+coverage/
+**__fixtures__**
+
+src/common/tabs/tab.js
```

Comment:
- These files are inherited and not developed by us recently. 
- Most of them is dead code. I'm not even sure why do we still keep them.
- I can fix them right away, but in that case I would like to have the code coverage turned off until the fix will be merged. (I don't want to write test for them just because I fixed some linting error. I can not understand them so fast, to be able to cover them with tests.) 
- They are very poorly written, I am wondering if some of the code even runs. 

For example this one:
![image](https://github.com/user-attachments/assets/81cce531-3bfa-4702-afad-3e27c8df38c3)

and it is used inside this component (see it below) which is not used at all.
![image](https://github.com/user-attachments/assets/e81cbdab-0050-48fd-ac2e-fbb5cd74a487)

Reply:
okay, I guess that code would blow up the linting tool  :)

Reply:
Here is another example:
![image](https://github.com/user-attachments/assets/7a8aaad9-06f7-4aec-bfef-d99f3a73c1e5)
 
 This would be the better way to do it: 
![image](https://github.com/user-attachments/assets/258d473e-d7d1-48d4-b7c5-7e66cbfa22ed)